Thorn/theta-start.sh sv-benchmarks/c/array-examples/sanfoundry_24-1.i --svcomp --portfolio HORN --loglevel RESULT --property sv-benchmarks/c/properties/unreach-call.prp --architecture ILP32


--------------------------------------------------------------------------------


Verifying input 'sv-benchmarks/c/array-examples/sanfoundry_24-1.i' with property 'sv-benchmarks/c/properties/unreach-call.prp' using arguments '--svcomp --portfolio HORN --loglevel RESULT --architecture ILP32'
LD_LIBRARY_PATH=/home/runner/work/theta/theta/Thorn/lib java -Xss120m -Xmx14210m -jar /home/runner/work/theta/theta/Thorn/theta.jar --svcomp --portfolio HORN --loglevel RESULT --architecture ILP32 --input sv-benchmarks/c/array-examples/sanfoundry_24-1.i --property sv-benchmarks/c/properties/unreach-call.prp --smt-home /home/runner/work/theta/theta/Thorn/solvers
Arithmetic: [ARR, NONLIN_INT]
ParsingResult Success
Alias graph size: 1 -> [2]
Using CHC portfolio
Current configuration: XcfaConfig(inputConfig=InputConfig(inputFile=null, catFile=null, parseCtx=null, xcfaWCtx=present, propertyFile=null, property=ERROR_LOCATION, frontendConfig=FrontendConfig(lbeLevel=LBE_SEQ, staticCoi=false, loopUnroll=1000, forceUnroll=-1, enableFew=false, inputType=C, specConfig=CFrontendConfig(arithmetic=efficient, architecture=LP64)), backendConfig=BackendConfig(backend=CHC, solverHome=/home/runner/work/theta/theta/Thorn/solvers, timeoutMs=100000, inProcess=true, parseInProcess=false, memlimit=0, specConfig=HornConfig(solver=z3:4.13.0, validateSolver=false, rankingFuncConstr=ADD)), outputConfig=OutputConfig(versionInfo=false, enableOutput=true, resultFolder=., acceptUnreliableSafe=false, cOutputConfig=COutputConfig(disable=true, useArr=false, useExArr=false, useRange=false), xcfaOutputConfig=XcfaOutputConfig(disable=false), chcOutputConfig=ChcOutputConfig(disable=false), witnessConfig=WitnessConfig(disable=false, svcomp=false, concretizerSolver=Z3, validateConcretizerSolver=false, inputFileForWitness=sv-benchmarks/c/array-examples/sanfoundry_24-1.i), argConfig=ArgConfig(disable=true)), debugConfig=DebugConfig(debug=false, stacktrace=false, logLevel=RESULT, argdebug=false, argToFile=false))
server: Process failed! (hu.bme.mit.theta.solver.smtlib.solver.SmtLibSolver.issueGeneralCommand(SmtLibSolver.java:352), hu.bme.mit.theta.solver.smtlib.solver.SmtLibSolverException: "line 6 column 20: invalid sorted variables: unknown sort 'printEven'")
ErrorCodeException(code=221)
	at hu.bme.mit.theta.xcfa.cli.checkers.InProcessChecker.check(InProcessChecker.kt:144)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:260)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.params.ExitCodesKt.exitOnError(ExitCodes.kt:56)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.backend(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.runConfig(ExecuteConfig.kt:92)
	at hu.bme.mit.theta.xcfa.cli.portfolio.Horn25Kt$hornPortfolio25$checker$1.invoke(horn25.kt:39)
	at hu.bme.mit.theta.xcfa.cli.portfolio.Horn25Kt$hornPortfolio25$checker$1.invoke(horn25.kt:39)
	at hu.bme.mit.theta.xcfa.cli.portfolio.ConfigNode.execute(stm.kt:70)
	at hu.bme.mit.theta.xcfa.cli.portfolio.STM.execute(stm.kt:154)
	at hu.bme.mit.theta.xcfa.cli.portfolio.HierarchicalNode.execute(stm.kt:35)
	at hu.bme.mit.theta.xcfa.cli.portfolio.STM.execute(stm.kt:154)
	at hu.bme.mit.theta.xcfa.cli.checkers.ConfigToPortfolioKt.getPortfolioChecker$lambda$0(ConfigToPortfolio.kt:106)
	at hu.bme.mit.theta.analysis.algorithm.SafetyChecker.check(SafetyChecker.java:27)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:260)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.params.ExitCodesKt.exitOnError(ExitCodes.kt:56)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.backend(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.runConfig(ExecuteConfig.kt:92)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli.run(XcfaCli.kt:127)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli.access$run(XcfaCli.kt:37)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli$Companion.main(XcfaCli.kt:137)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli.main(XcfaCli.kt)
Caught exception: ErrorCodeException(code=221)
Handling exception as Anything
Current configuration: XcfaConfig(inputConfig=InputConfig(inputFile=null, catFile=null, parseCtx=null, xcfaWCtx=present, propertyFile=null, property=ERROR_LOCATION, frontendConfig=FrontendConfig(lbeLevel=LBE_SEQ, staticCoi=false, loopUnroll=1000, forceUnroll=-1, enableFew=false, inputType=C, specConfig=CFrontendConfig(arithmetic=efficient, architecture=LP64)), backendConfig=BackendConfig(backend=CHC, solverHome=/home/runner/work/theta/theta/Thorn/solvers, timeoutMs=100000, inProcess=true, parseInProcess=false, memlimit=0, specConfig=HornConfig(solver=Z3:4.13, validateSolver=false, rankingFuncConstr=ADD)), outputConfig=OutputConfig(versionInfo=false, enableOutput=true, resultFolder=., acceptUnreliableSafe=false, cOutputConfig=COutputConfig(disable=true, useArr=false, useExArr=false, useRange=false), xcfaOutputConfig=XcfaOutputConfig(disable=false), chcOutputConfig=ChcOutputConfig(disable=false), witnessConfig=WitnessConfig(disable=false, svcomp=false, concretizerSolver=Z3, validateConcretizerSolver=false, inputFileForWitness=sv-benchmarks/c/array-examples/sanfoundry_24-1.i), argConfig=ArgConfig(disable=true)), debugConfig=DebugConfig(debug=false, stacktrace=false, logLevel=RESULT, argdebug=false, argToFile=false))
server: Exception in thread "main" com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: java.lang.AssertionError: Undefined default case executed for: (deref |main::array| (mod |main::i| 4294967296) Int)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2053)
server: 	at com.google.common.cache.LocalCache.get(LocalCache.java:3966)
server: 	at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4863)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.toTerm(Z3ExprTransformer.java:229)
server: 	at hu.bme.mit.theta.solver.z3.Z3TransformationManager.toTerm(Z3TransformationManager.java:44)
server: 	at hu.bme.mit.theta.solver.z3.Z3Solver.add(Z3Solver.java:93)
server: 	at hu.bme.mit.theta.solver.Solver.add(Solver.java:46)
server: 	at hu.bme.mit.theta.solver.HornSolver.add(HornSolver.java:30)
server: 	at hu.bme.mit.theta.analysis.algorithm.chc.HornChecker.check(HornChecker.kt:49)
server: 	at hu.bme.mit.theta.xcfa.cli.checkers.ConfigToHornCheckerKt.getHornChecker$lambda$0(ConfigToHornChecker.kt:83)
server: 	at hu.bme.mit.theta.analysis.algorithm.SafetyChecker.check(SafetyChecker.java:27)
server: 	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:260)
server: 	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:259)
server: 	at hu.bme.mit.theta.xcfa.cli.params.ExitCodesKt.exitOnError(ExitCodes.kt:56)
server: 	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.backend(ExecuteConfig.kt:259)
server: 	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.runConfig(ExecuteConfig.kt:92)
server: 	at hu.bme.mit.theta.xcfa.cli.XcfaCli.run(XcfaCli.kt:127)
server: 	at hu.bme.mit.theta.xcfa.cli.XcfaCli.access$run(XcfaCli.kt:37)
server: 	at hu.bme.mit.theta.xcfa.cli.XcfaCli$Companion.main(XcfaCli.kt:137)
server: 	at hu.bme.mit.theta.xcfa.cli.XcfaCli.main(XcfaCli.kt)
server: Caused by: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: java.lang.AssertionError: Undefined default case executed for: (deref |main::array| (mod |main::i| 4294967296) Int)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2053)
server: 	at com.google.common.cache.LocalCache.get(LocalCache.java:3966)
server: 	at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4863)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.toTerm(Z3ExprTransformer.java:229)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.transformForall(Z3ExprTransformer.java:316)
server: 	at hu.bme.mit.theta.common.DispatchTable.dispatch(DispatchTable.java:43)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.lambda$toTerm$0(Z3ExprTransformer.java:229)
server: 	at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4868)
server: 	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3533)
server: 	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2282)
server: 	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2159)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2049)
server: 	... 19 more
server: Caused by: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: java.lang.AssertionError: Undefined default case executed for: (deref |main::array| (mod |main::i| 4294967296) Int)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2053)
server: 	at com.google.common.cache.LocalCache.get(LocalCache.java:3966)
server: 	at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4863)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.toTerm(Z3ExprTransformer.java:229)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.transformImply(Z3ExprTransformer.java:275)
server: 	at hu.bme.mit.theta.common.DispatchTable.dispatch(DispatchTable.java:43)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.lambda$toTerm$0(Z3ExprTransformer.java:229)
server: 	at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4868)
server: 	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3533)
server: 	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2282)
server: 	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2159)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2049)
server: 	... 30 more
server: Caused by: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: java.lang.AssertionError: Undefined default case executed for: (deref |main::array| (mod |main::i| 4294967296) Int)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2053)
server: 	at com.google.common.cache.LocalCache.get(LocalCache.java:3966)
server: 	at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4863)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.toTerm(Z3ExprTransformer.java:229)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.lambda$transformAnd$1(Z3ExprTransformer.java:294)
server: 	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
server: 	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:992)
server: 	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
server: 	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
server: 	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575)
server: 	at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260)
server: 	at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.transformAnd(Z3ExprTransformer.java:294)
server: 	at hu.bme.mit.theta.common.DispatchTable.dispatch(DispatchTable.java:43)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.lambda$toTerm$0(Z3ExprTransformer.java:229)
server: 	at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4868)
server: 	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3533)
server: 	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2282)
server: 	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2159)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2049)
server: 	... 41 more
server: Caused by: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: java.lang.AssertionError: Undefined default case executed for: (deref |main::array| (mod |main::i| 4294967296) Int)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2053)
server: 	at com.google.common.cache.LocalCache.get(LocalCache.java:3966)
server: 	at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4863)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.toTerm(Z3ExprTransformer.java:229)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.lambda$transformAnd$1(Z3ExprTransformer.java:294)
server: 	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
server: 	at java.base/java.util.Collections$2.tryAdvance(Collections.java:4854)
server: 	at java.base/java.util.Collections$2.forEachRemaining(Collections.java:4862)
server: 	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
server: 	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
server: 	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575)
server: 	at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260)
server: 	at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.transformAnd(Z3ExprTransformer.java:294)
server: 	at hu.bme.mit.theta.common.DispatchTable.dispatch(DispatchTable.java:43)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.lambda$toTerm$0(Z3ExprTransformer.java:229)
server: 	at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4868)
server: 	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3533)
server: 	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2282)
server: 	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2159)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2049)
server: 	... 60 more
server: Caused by: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: java.lang.AssertionError: Undefined default case executed for: (deref |main::array| (mod |main::i| 4294967296) Int)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2053)
server: 	at com.google.common.cache.LocalCache.get(LocalCache.java:3966)
server: 	at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4863)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.toTerm(Z3ExprTransformer.java:229)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.lambda$transformAnd$1(Z3ExprTransformer.java:294)
server: 	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
server: 	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:992)
server: 	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
server: 	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
server: 	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575)
server: 	at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260)
server: 	at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.transformAnd(Z3ExprTransformer.java:294)
server: 	at hu.bme.mit.theta.common.DispatchTable.dispatch(DispatchTable.java:43)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.lambda$toTerm$0(Z3ExprTransformer.java:229)
server: 	at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4868)
server: 	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3533)
server: 	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2282)
server: 	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2159)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2049)
server: 	... 80 more
server: Caused by: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: java.lang.AssertionError: Undefined default case executed for: (deref |main::array| (mod |main::i| 4294967296) Int)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2053)
server: 	at com.google.common.cache.LocalCache.get(LocalCache.java:3966)
server: 	at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4863)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.toTerm(Z3ExprTransformer.java:229)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.transformNot(Z3ExprTransformer.java:270)
server: 	at hu.bme.mit.theta.common.DispatchTable.dispatch(DispatchTable.java:43)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.lambda$toTerm$0(Z3ExprTransformer.java:229)
server: 	at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4868)
server: 	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3533)
server: 	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2282)
server: 	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2159)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2049)
server: 	... 99 more
server: Caused by: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: java.lang.AssertionError: Undefined default case executed for: (deref |main::array| (mod |main::i| 4294967296) Int)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2053)
server: 	at com.google.common.cache.LocalCache.get(LocalCache.java:3966)
server: 	at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4863)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.toTerm(Z3ExprTransformer.java:229)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.transformIntEq(Z3ExprTransformer.java:510)
server: 	at hu.bme.mit.theta.common.DispatchTable.dispatch(DispatchTable.java:43)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.lambda$toTerm$0(Z3ExprTransformer.java:229)
server: 	at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4868)
server: 	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3533)
server: 	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2282)
server: 	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2159)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2049)
server: 	... 110 more
server: Caused by: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: java.lang.AssertionError: Undefined default case executed for: (deref |main::array| (mod |main::i| 4294967296) Int)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2053)
server: 	at com.google.common.cache.LocalCache.get(LocalCache.java:3966)
server: 	at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4863)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.toTerm(Z3ExprTransformer.java:229)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.transformIte(Z3ExprTransformer.java:255)
server: 	at hu.bme.mit.theta.common.DispatchTable.dispatch(DispatchTable.java:43)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.lambda$toTerm$0(Z3ExprTransformer.java:229)
server: 	at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4868)
server: 	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3533)
server: 	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2282)
server: 	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2159)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2049)
server: 	... 121 more
server: Caused by: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: java.lang.AssertionError: Undefined default case executed for: (deref |main::array| (mod |main::i| 4294967296) Int)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2053)
server: 	at com.google.common.cache.LocalCache.get(LocalCache.java:3966)
server: 	at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4863)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.toTerm(Z3ExprTransformer.java:229)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.transformIntEq(Z3ExprTransformer.java:510)
server: 	at hu.bme.mit.theta.common.DispatchTable.dispatch(DispatchTable.java:43)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.lambda$toTerm$0(Z3ExprTransformer.java:229)
server: 	at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4868)
server: 	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3533)
server: 	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2282)
server: 	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2159)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2049)
server: 	... 132 more
server: Caused by: com.google.common.util.concurrent.ExecutionError: java.lang.AssertionError: Undefined default case executed for: (deref |main::array| (mod |main::i| 4294967296) Int)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2053)
server: 	at com.google.common.cache.LocalCache.get(LocalCache.java:3966)
server: 	at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4863)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.toTerm(Z3ExprTransformer.java:229)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.transformIntMod(Z3ExprTransformer.java:495)
server: 	at hu.bme.mit.theta.common.DispatchTable.dispatch(DispatchTable.java:43)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.lambda$toTerm$0(Z3ExprTransformer.java:229)
server: 	at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4868)
server: 	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3533)
server: 	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2282)
server: 	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2159)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2049)
server: 	... 143 more
server: Caused by: java.lang.AssertionError: Undefined default case executed for: (deref |main::array| (mod |main::i| 4294967296) Int)
server: 	at hu.bme.mit.theta.common.DispatchTable$Builder.lambda$build$0(DispatchTable.java:88)
server: 	at hu.bme.mit.theta.common.DispatchTable.dispatch(DispatchTable.java:41)
server: 	at hu.bme.mit.theta.solver.z3.Z3ExprTransformer.lambda$toTerm$0(Z3ExprTransformer.java:229)
server: 	at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4868)
server: 	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3533)
server: 	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2282)
server: 	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2159)
server: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2049)
server: 	... 154 more
ErrorCodeException(code=1)
	at hu.bme.mit.theta.xcfa.cli.checkers.InProcessChecker.check(InProcessChecker.kt:144)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:260)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.params.ExitCodesKt.exitOnError(ExitCodes.kt:56)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.backend(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.runConfig(ExecuteConfig.kt:92)
	at hu.bme.mit.theta.xcfa.cli.portfolio.Horn25Kt$hornPortfolio25$checker$1.invoke(horn25.kt:39)
	at hu.bme.mit.theta.xcfa.cli.portfolio.Horn25Kt$hornPortfolio25$checker$1.invoke(horn25.kt:39)
	at hu.bme.mit.theta.xcfa.cli.portfolio.ConfigNode.execute(stm.kt:70)
	at hu.bme.mit.theta.xcfa.cli.portfolio.STM.execute(stm.kt:154)
	at hu.bme.mit.theta.xcfa.cli.portfolio.HierarchicalNode.execute(stm.kt:35)
	at hu.bme.mit.theta.xcfa.cli.portfolio.STM.execute(stm.kt:154)
	at hu.bme.mit.theta.xcfa.cli.checkers.ConfigToPortfolioKt.getPortfolioChecker$lambda$0(ConfigToPortfolio.kt:106)
	at hu.bme.mit.theta.analysis.algorithm.SafetyChecker.check(SafetyChecker.java:27)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:260)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.params.ExitCodesKt.exitOnError(ExitCodes.kt:56)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.backend(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.runConfig(ExecuteConfig.kt:92)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli.run(XcfaCli.kt:127)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli.access$run(XcfaCli.kt:37)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli$Companion.main(XcfaCli.kt:137)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli.main(XcfaCli.kt)
Caught exception: ErrorCodeException(code=1)
Handling exception as Anything
Current configuration: XcfaConfig(inputConfig=InputConfig(inputFile=null, catFile=null, parseCtx=null, xcfaWCtx=present, propertyFile=null, property=ERROR_LOCATION, frontendConfig=FrontendConfig(lbeLevel=LBE_SEQ, staticCoi=false, loopUnroll=1000, forceUnroll=-1, enableFew=false, inputType=C, specConfig=CFrontendConfig(arithmetic=efficient, architecture=LP64)), backendConfig=BackendConfig(backend=CHC, solverHome=/home/runner/work/theta/theta/Thorn/solvers, timeoutMs=500000, inProcess=true, parseInProcess=false, memlimit=0, specConfig=HornConfig(solver=eldarica:2.1, validateSolver=false, rankingFuncConstr=ADD)), outputConfig=OutputConfig(versionInfo=false, enableOutput=true, resultFolder=., acceptUnreliableSafe=false, cOutputConfig=COutputConfig(disable=true, useArr=false, useExArr=false, useRange=false), xcfaOutputConfig=XcfaOutputConfig(disable=false), chcOutputConfig=ChcOutputConfig(disable=false), witnessConfig=WitnessConfig(disable=false, svcomp=false, concretizerSolver=Z3, validateConcretizerSolver=false, inputFileForWitness=sv-benchmarks/c/array-examples/sanfoundry_24-1.i), argConfig=ArgConfig(disable=true)), debugConfig=DebugConfig(debug=false, stacktrace=false, logLevel=RESULT, argdebug=false, argToFile=false))
server: Process failed! (hu.bme.mit.theta.solver.smtlib.impl.generic.GenericSmtLibExprTransformer.toTerm(GenericSmtLibExprTransformer.java:336), com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: java.lang.IllegalStateException: Incomplete dereferences (missing uniquenessIdx) are not handled properly.)
ErrorCodeException(code=202)
	at hu.bme.mit.theta.xcfa.cli.checkers.InProcessChecker.check(InProcessChecker.kt:144)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:260)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.params.ExitCodesKt.exitOnError(ExitCodes.kt:56)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.backend(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.runConfig(ExecuteConfig.kt:92)
	at hu.bme.mit.theta.xcfa.cli.portfolio.Horn25Kt$hornPortfolio25$checker$1.invoke(horn25.kt:39)
	at hu.bme.mit.theta.xcfa.cli.portfolio.Horn25Kt$hornPortfolio25$checker$1.invoke(horn25.kt:39)
	at hu.bme.mit.theta.xcfa.cli.portfolio.ConfigNode.execute(stm.kt:70)
	at hu.bme.mit.theta.xcfa.cli.portfolio.STM.execute(stm.kt:154)
	at hu.bme.mit.theta.xcfa.cli.portfolio.HierarchicalNode.execute(stm.kt:35)
	at hu.bme.mit.theta.xcfa.cli.portfolio.STM.execute(stm.kt:154)
	at hu.bme.mit.theta.xcfa.cli.checkers.ConfigToPortfolioKt.getPortfolioChecker$lambda$0(ConfigToPortfolio.kt:106)
	at hu.bme.mit.theta.analysis.algorithm.SafetyChecker.check(SafetyChecker.java:27)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:260)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.params.ExitCodesKt.exitOnError(ExitCodes.kt:56)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.backend(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.runConfig(ExecuteConfig.kt:92)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli.run(XcfaCli.kt:127)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli.access$run(XcfaCli.kt:37)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli$Companion.main(XcfaCli.kt:137)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli.main(XcfaCli.kt)
Caught exception: ErrorCodeException(code=202)
Handling exception as Anything
Current configuration: XcfaConfig(inputConfig=InputConfig(inputFile=null, catFile=null, parseCtx=null, xcfaWCtx=present, propertyFile=null, property=ERROR_LOCATION, frontendConfig=FrontendConfig(lbeLevel=LBE_SEQ, staticCoi=false, loopUnroll=1000, forceUnroll=-1, enableFew=false, inputType=C, specConfig=CFrontendConfig(arithmetic=efficient, architecture=LP64)), backendConfig=BackendConfig(backend=CHC, solverHome=/home/runner/work/theta/theta/Thorn/solvers, timeoutMs=300000, inProcess=true, parseInProcess=false, memlimit=0, specConfig=HornConfig(solver=golem:0.5.0, validateSolver=false, rankingFuncConstr=ADD)), outputConfig=OutputConfig(versionInfo=false, enableOutput=true, resultFolder=., acceptUnreliableSafe=false, cOutputConfig=COutputConfig(disable=true, useArr=false, useExArr=false, useRange=false), xcfaOutputConfig=XcfaOutputConfig(disable=false), chcOutputConfig=ChcOutputConfig(disable=false), witnessConfig=WitnessConfig(disable=false, svcomp=false, concretizerSolver=Z3, validateConcretizerSolver=false, inputFileForWitness=sv-benchmarks/c/array-examples/sanfoundry_24-1.i), argConfig=ArgConfig(disable=true)), debugConfig=DebugConfig(debug=false, stacktrace=false, logLevel=RESULT, argdebug=false, argToFile=false))
server: Process failed! (hu.bme.mit.theta.solver.smtlib.impl.generic.GenericSmtLibExprTransformer.toTerm(GenericSmtLibExprTransformer.java:336), com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: java.lang.IllegalStateException: Incomplete dereferences (missing uniquenessIdx) are not handled properly.)
ErrorCodeException(code=202)
	at hu.bme.mit.theta.xcfa.cli.checkers.InProcessChecker.check(InProcessChecker.kt:144)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:260)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.params.ExitCodesKt.exitOnError(ExitCodes.kt:56)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.backend(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.runConfig(ExecuteConfig.kt:92)
	at hu.bme.mit.theta.xcfa.cli.portfolio.Horn25Kt$hornPortfolio25$checker$1.invoke(horn25.kt:39)
	at hu.bme.mit.theta.xcfa.cli.portfolio.Horn25Kt$hornPortfolio25$checker$1.invoke(horn25.kt:39)
	at hu.bme.mit.theta.xcfa.cli.portfolio.ConfigNode.execute(stm.kt:70)
	at hu.bme.mit.theta.xcfa.cli.portfolio.STM.execute(stm.kt:154)
	at hu.bme.mit.theta.xcfa.cli.portfolio.HierarchicalNode.execute(stm.kt:35)
	at hu.bme.mit.theta.xcfa.cli.portfolio.STM.execute(stm.kt:154)
	at hu.bme.mit.theta.xcfa.cli.checkers.ConfigToPortfolioKt.getPortfolioChecker$lambda$0(ConfigToPortfolio.kt:106)
	at hu.bme.mit.theta.analysis.algorithm.SafetyChecker.check(SafetyChecker.java:27)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:260)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.params.ExitCodesKt.exitOnError(ExitCodes.kt:56)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.backend(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.runConfig(ExecuteConfig.kt:92)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli.run(XcfaCli.kt:127)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli.access$run(XcfaCli.kt:37)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli$Companion.main(XcfaCli.kt:137)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli.main(XcfaCli.kt)
Caught exception: ErrorCodeException(code=202)
Could not handle trigger ErrorCodeException(code=202) (Available triggers: [])
Caught exception: ErrorCodeException(code=202)
Handling exception as Anything
Current configuration: XcfaConfig(inputConfig=InputConfig(inputFile=null, catFile=null, parseCtx=null, xcfaWCtx=present, propertyFile=null, property=ERROR_LOCATION, frontendConfig=FrontendConfig(lbeLevel=LBE_SEQ, staticCoi=false, loopUnroll=1000, forceUnroll=-1, enableFew=false, inputType=C, specConfig=CFrontendConfig(arithmetic=efficient, architecture=LP64)), backendConfig=BackendConfig(backend=CHC, solverHome=/home/runner/work/theta/theta/Thorn/solvers, timeoutMs=100000, inProcess=false, parseInProcess=false, memlimit=0, specConfig=HornConfig(solver=z3:4.13.0, validateSolver=false, rankingFuncConstr=ADD)), outputConfig=OutputConfig(versionInfo=false, enableOutput=true, resultFolder=., acceptUnreliableSafe=false, cOutputConfig=COutputConfig(disable=true, useArr=false, useExArr=false, useRange=false), xcfaOutputConfig=XcfaOutputConfig(disable=false), chcOutputConfig=ChcOutputConfig(disable=false), witnessConfig=WitnessConfig(disable=false, svcomp=false, concretizerSolver=Z3, validateConcretizerSolver=false, inputFileForWitness=sv-benchmarks/c/array-examples/sanfoundry_24-1.i), argConfig=ArgConfig(disable=true)), debugConfig=DebugConfig(debug=false, stacktrace=false, logLevel=RESULT, argdebug=false, argToFile=false))
Process failed! (hu.bme.mit.theta.solver.smtlib.solver.SmtLibSolver.issueGeneralCommand(SmtLibSolver.java:352), hu.bme.mit.theta.solver.smtlib.solver.SmtLibSolverException: "line 6 column 20: invalid sorted variables: unknown sort 'printEven'")
Caught exception: ErrorCodeException(code=221)
Handling exception as Anything
Current configuration: XcfaConfig(inputConfig=InputConfig(inputFile=null, catFile=null, parseCtx=null, xcfaWCtx=present, propertyFile=null, property=ERROR_LOCATION, frontendConfig=FrontendConfig(lbeLevel=LBE_SEQ, staticCoi=false, loopUnroll=1000, forceUnroll=-1, enableFew=false, inputType=C, specConfig=CFrontendConfig(arithmetic=efficient, architecture=LP64)), backendConfig=BackendConfig(backend=CHC, solverHome=/home/runner/work/theta/theta/Thorn/solvers, timeoutMs=100000, inProcess=false, parseInProcess=false, memlimit=0, specConfig=HornConfig(solver=Z3:4.13, validateSolver=false, rankingFuncConstr=ADD)), outputConfig=OutputConfig(versionInfo=false, enableOutput=true, resultFolder=., acceptUnreliableSafe=false, cOutputConfig=COutputConfig(disable=true, useArr=false, useExArr=false, useRange=false), xcfaOutputConfig=XcfaOutputConfig(disable=false), chcOutputConfig=ChcOutputConfig(disable=false), witnessConfig=WitnessConfig(disable=false, svcomp=false, concretizerSolver=Z3, validateConcretizerSolver=false, inputFileForWitness=sv-benchmarks/c/array-examples/sanfoundry_24-1.i), argConfig=ArgConfig(disable=true)), debugConfig=DebugConfig(debug=false, stacktrace=false, logLevel=RESULT, argdebug=false, argToFile=false))
Caught exception: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: java.lang.AssertionError: Undefined default case executed for: (deref |main::array| (mod |main::i| 4294967296) Int)
Handling exception as Anything
Current configuration: XcfaConfig(inputConfig=InputConfig(inputFile=null, catFile=null, parseCtx=null, xcfaWCtx=present, propertyFile=null, property=ERROR_LOCATION, frontendConfig=FrontendConfig(lbeLevel=LBE_SEQ, staticCoi=false, loopUnroll=1000, forceUnroll=-1, enableFew=false, inputType=C, specConfig=CFrontendConfig(arithmetic=efficient, architecture=LP64)), backendConfig=BackendConfig(backend=CHC, solverHome=/home/runner/work/theta/theta/Thorn/solvers, timeoutMs=500000, inProcess=false, parseInProcess=false, memlimit=0, specConfig=HornConfig(solver=eldarica:2.1, validateSolver=false, rankingFuncConstr=ADD)), outputConfig=OutputConfig(versionInfo=false, enableOutput=true, resultFolder=., acceptUnreliableSafe=false, cOutputConfig=COutputConfig(disable=true, useArr=false, useExArr=false, useRange=false), xcfaOutputConfig=XcfaOutputConfig(disable=false), chcOutputConfig=ChcOutputConfig(disable=false), witnessConfig=WitnessConfig(disable=false, svcomp=false, concretizerSolver=Z3, validateConcretizerSolver=false, inputFileForWitness=sv-benchmarks/c/array-examples/sanfoundry_24-1.i), argConfig=ArgConfig(disable=true)), debugConfig=DebugConfig(debug=false, stacktrace=false, logLevel=RESULT, argdebug=false, argToFile=false))
Process failed! (hu.bme.mit.theta.solver.smtlib.impl.generic.GenericSmtLibExprTransformer.toTerm(GenericSmtLibExprTransformer.java:336), com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: java.lang.IllegalStateException: Incomplete dereferences (missing uniquenessIdx) are not handled properly.)
Caught exception: ErrorCodeException(code=202)
Handling exception as Anything
Current configuration: XcfaConfig(inputConfig=InputConfig(inputFile=null, catFile=null, parseCtx=null, xcfaWCtx=present, propertyFile=null, property=ERROR_LOCATION, frontendConfig=FrontendConfig(lbeLevel=LBE_SEQ, staticCoi=false, loopUnroll=1000, forceUnroll=-1, enableFew=false, inputType=C, specConfig=CFrontendConfig(arithmetic=efficient, architecture=LP64)), backendConfig=BackendConfig(backend=CHC, solverHome=/home/runner/work/theta/theta/Thorn/solvers, timeoutMs=300000, inProcess=false, parseInProcess=false, memlimit=0, specConfig=HornConfig(solver=golem:0.5.0, validateSolver=false, rankingFuncConstr=ADD)), outputConfig=OutputConfig(versionInfo=false, enableOutput=true, resultFolder=., acceptUnreliableSafe=false, cOutputConfig=COutputConfig(disable=true, useArr=false, useExArr=false, useRange=false), xcfaOutputConfig=XcfaOutputConfig(disable=false), chcOutputConfig=ChcOutputConfig(disable=false), witnessConfig=WitnessConfig(disable=false, svcomp=false, concretizerSolver=Z3, validateConcretizerSolver=false, inputFileForWitness=sv-benchmarks/c/array-examples/sanfoundry_24-1.i), argConfig=ArgConfig(disable=true)), debugConfig=DebugConfig(debug=false, stacktrace=false, logLevel=RESULT, argdebug=false, argToFile=false))
Process failed! (hu.bme.mit.theta.solver.smtlib.impl.generic.GenericSmtLibExprTransformer.toTerm(GenericSmtLibExprTransformer.java:336), com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: com.google.common.util.concurrent.UncheckedExecutionException: java.lang.IllegalStateException: Incomplete dereferences (missing uniquenessIdx) are not handled properly.)
Caught exception: ErrorCodeException(code=202)
Could not handle trigger ErrorCodeException(code=202) (Available triggers: [])
Caught exception: ErrorCodeException(code=202)
Could not handle trigger ErrorCodeException(code=202) (Available triggers: [])
ErrorCodeException(code=202)
	at hu.bme.mit.theta.xcfa.cli.params.ExitCodesKt.exitProcess(ExitCodes.kt:44)
	at hu.bme.mit.theta.xcfa.cli.params.ExitCodesKt.exitOnError(ExitCodes.kt:94)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.backend(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.runConfig(ExecuteConfig.kt:92)
	at hu.bme.mit.theta.xcfa.cli.portfolio.Horn25Kt$hornPortfolio25$checker$1.invoke(horn25.kt:39)
	at hu.bme.mit.theta.xcfa.cli.portfolio.Horn25Kt$hornPortfolio25$checker$1.invoke(horn25.kt:39)
	at hu.bme.mit.theta.xcfa.cli.portfolio.ConfigNode.execute(stm.kt:70)
	at hu.bme.mit.theta.xcfa.cli.portfolio.STM.execute(stm.kt:154)
	at hu.bme.mit.theta.xcfa.cli.portfolio.HierarchicalNode.execute(stm.kt:35)
	at hu.bme.mit.theta.xcfa.cli.portfolio.STM.execute(stm.kt:154)
	at hu.bme.mit.theta.xcfa.cli.checkers.ConfigToPortfolioKt.getPortfolioChecker$lambda$0(ConfigToPortfolio.kt:106)
	at hu.bme.mit.theta.analysis.algorithm.SafetyChecker.check(SafetyChecker.java:27)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:260)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt$backend$result$1.invoke(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.params.ExitCodesKt.exitOnError(ExitCodes.kt:56)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.backend(ExecuteConfig.kt:259)
	at hu.bme.mit.theta.xcfa.cli.ExecuteConfigKt.runConfig(ExecuteConfig.kt:92)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli.run(XcfaCli.kt:127)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli.access$run(XcfaCli.kt:37)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli$Companion.main(XcfaCli.kt:137)
	at hu.bme.mit.theta.xcfa.cli.XcfaCli.main(XcfaCli.kt)
